### DISCUSSION TITLE
Add search box to field selection in data model

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#14466] [Feature Requests]
When setting up new fields in a collection, I keep trying to find the correct interface and end up using the cmd+f shortcut in the browser, which isn't that great of an experience. It would be great if there was a search box on the top right of the drawer that's auto-focussed so I can just type in the name of the interface (like `file`)

**Comment by rijkvanzanten** [comment]
I think @w0kyj mentioned this earlier as well

**Comment by w0kyj** [comment]
Would also be really nice to have a search on:
- Data Model Collections
- Collections - Fields

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
